keras_core/utils/summary_utils.py

```diff
@@ -349,11 +351,13 @@ def print_layer(layer, nested_level=0):
         + highlight_number(f"{non_trainable_count:,}")
         + f" ({readable_memory_size(non_trainable_memory_size)})"
     )
-    console.print(
-        bold_text(" Optimizer params: ")
-        + highlight_number(f"{optimizer_weight_count:,}")
-        + f" ({readable_memory_size(optimizer_memory_size)})"
-    )
+    if optimizer_built:
+        console.print(
+            bold_text(" Optimizer params: ")
+            + highlight_number(f"{optimizer_weight_count:,}")
+            + f" ({readable_memory_size(optimizer_memory_size)})"
+            + ("" if optimizer_built else " (unbuilt)")
```

Comment:
The if condition seems always be True here since the l354 is true.

Reply:
oops! good catch, mean to remove this